assembl/static2/css/components/navbar.scss

```diff
@@ -320,3 +320,11 @@
     }
   }
 }
+
+.large-logo {
+  padding: 25px 0 0 20px;
+
+  img {
+    max-height: 60px;
```

Comment:
@swergas Ici j'ai mis une hauteur maximale pour que le logo ne dépasse pas.

Reply:
Oui mais c'est que pour le cas large-logo on dirait ? Ma question était qu'en est-il pour le cas où tout est sur la même ligne ?

Reply:
Should this value change in accordance with different screen resolutions?

Reply:
nope